src/aind_metadata_service/__init__.py

```diff
@@ -1,3 +1,3 @@
 """REST service to retrieve metadata from databases.
 """
-__version__ = "0.1.1"
+__version__ = "0.1.2"
```

Comment:
This no longer needs to be updated. We added a feature to automatically manage the semantic version bumps.

Reply:
Oh nice! should i change it back to 0.1.1 or leave it

Reply:
Go ahead and revert it back to 0.1.1